### PR TITLE
DOCS-9429 Adding proxy info for BYOK / Revising links to RTFCTL links

### PR DESCRIPTION
**Pull request by sleslie23**
Adding all remaining RTF tasks to this PR.

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -17,9 +18,10 @@
  ** xref:configure-adv-tls-context.adoc[Import a TLS Context from Secrets Manager (Advanced)] 
  ** xref:runtime-fabric-logs.adoc[View and Configure Logs in Runtime Fabric]
  ** xref:configure-alerting.adoc[Configure Alerting on Anypoint Runtime Fabric]
+ ** xref:using-opscenter.adoc[Use Ops Center to Monitor Runtime Fabric on VMs / Bare Metal]
  ** xref:troubleshoot-guide.adoc[Troubleshooting Runtime Fabric]
- ** xref:upgrade-index.adoc[Upgrade Runtime Fabric]
- ** xref:uninstall-manual.adoc[Uninstall Runtime Fabric Manually]
+ ** xref:upgrade-index.adoc[Upgrade Runtime Fabric on VMs / Bare Metal]
+ ** xref:uninstall-manual.adoc[Uninstall Upgrade Runtime Fabric on VMs / Bare Metal]
```
Comment (sarjeet2013) [inline modules/ROOT/nav.adoc]: Uninstall Upgrade?

File: modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -32,11 +34,11 @@
 * xref:manage-index.adoc[Manage Runtime Fabric]
  ** xref:view-health.adoc[View Runtime Fabric Health Details]
  ** xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric on VMs / Bare Metal]
- ** xref:using-opscenter.adoc[Monitor Anypoint Runtime Fabric]
  ** xref:install-rtfctl.adoc[Install the Runtime Fabric Command Line Tool]
  ** xref:install-patches.adoc[Perform Scheduled Maintenance on Runtime Fabric on VMs / Bare Metal]
  ** xref:replace-license-key.adoc[Replace a Mule License Key]
- ** xref:manage-proxy.adoc[Manage Proxies used by Runtime Fabric]
+ ** xref:manage-proxy-self.adoc[Manage Proxies on Runtime Fabric on Self-Managed Kubernetes]
+ ** xref:manage-proxy.adoc[Manage Proxies Used by Runtime Fabric on VMs / Bare Metal]
```
Comment (sarjeet2013) [inline modules/ROOT/nav.adoc]: how about keep it consistent to either `Manage Proxies on`or `Manage Proxies Used by` from line above?

File: modules/ROOT/pages/manage-proxy-self.adoc [hunk 3]
```diff
@@ -0,0 +1,69 @@
+= Manage Proxies on Runtime Fabric on Self-Managed Kubernetes
```
Comment (sarjeet2013) [inline modules/ROOT/pages/manage-proxy-self.adoc:1]: Same comment as above. Keep it consistent to either `Used by` or `on`.

File: modules/ROOT/pages/manage-proxy-self.adoc [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+[IMPORTANT]
+====
+The procedures in this topic should be performed by a MuleSoft organization administrator.
+====
+
+Runtime Fabric on Self-Managed Kubernetes uses two types of proxies when running and hosting applications:
+
+* System proxies: include proxies that enable connectivity among Kubernetes cluster services, including the Docker daemon and other internal components. When using Runtime Fabric on Self-Managed Kubernetes you are responsible for configuring and managing system proxies.
+* Application proxies: include proxies that that enable connectivity to applications running on Runtime Fabric.
```
Comment (sarjeet2013) [inline modules/ROOT/pages/manage-proxy-self.adoc]: nit: Additional `that`

**Review by sarjeet2013 — APPROVED** [review]
Few minor comments, otherwise LGTM

File: modules/ROOT/nav.adoc (hunk 1 above)
Comment (sleslie23) [reply modules/ROOT/nav.adoc]: Fixed.

File: modules/ROOT/nav.adoc (hunk 2 above)
Comment (sleslie23) [reply modules/ROOT/nav.adoc]: manage proxies on

File: modules/ROOT/pages/manage-proxy-self.adoc (hunk 3 above)
Comment (sleslie23) [reply modules/ROOT/pages/manage-proxy-self.adoc:1]: Fixed.

File: modules/ROOT/pages/manage-proxy-self.adoc (hunk 4 above)
Comment (sleslie23) [reply modules/ROOT/pages/manage-proxy-self.adoc]: Fixed

Comment (sleslie23) [reply modules/ROOT/pages/manage-proxy-self.adoc]: Fixed

**Pull request merged**